<launch>
    <param name="roadmap_file" textfile="$(find mpc_traj_follower)/TestTrack1.csv" />
    <param name="distance_can_see" type="double" value="20.0"/>
    <param name="integration_dt" type="double" value="0.1"/>
    <param name="mpc_horizon" type="double" value="1"/>

    <param name="pos_x" type="double" value="5.55"/>
    <param name="pos_y" type="double" value="0.0"/>
    <param name="vel" type="double" value="0.0"/>
    <param name="yaw" type="double" value="1.570796327"/>
    
    <node name="plot"                pkg="mpc_traj_follower"   type="plot_vehicle.py"          output="screen"/>
    <node name="perception_node"     pkg="mpc_traj_follower"   type="perception"               output="screen"/>
    <node name="vehicle_plant_node"  pkg="mpc_traj_follower"   type="vehicle_plant_model"      output="screen"/>
    <node name="plan_control_node"   pkg="mpc_traj_follower"   type="track1_pnc_continuous.py" output="screen" cwd="node"/>
</launch>
